Source/WebCore/xml/parser/XMLDocumentParserLibxml2.cpp

```diff
@@ -1255,17 +1255,17 @@ static void internalSubsetHandler(void* closure, const xmlChar* name, const xmlC
 static void externalSubsetHandler(void* closure, const xmlChar*, const xmlChar* externalId, const xmlChar*)
 {
     String extId = toString(externalId);
```

Comment:
Should use `SortedArraySet` here, so many other places too. I’m not trying to comment on all of them.

Reply:
Will fix.